x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/build_events_query.ts

```diff
@@ -4,23 +4,36 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-interface BuildEventsScrollQuery {
+import { fromKueryExpression, toElasticsearchQuery } from '@kbn/es-query';
+
+interface BuildEventsSearchQuery {
   index: string[];
   from: string;
   to: string;
   filter: unknown;
   size: number;
-  scroll: string;
+  searchAfterSortId?: string;
 }
 
-export const buildEventsScrollQuery = ({
+export const getFilter = (kql: string | undefined, filter: Record<string, {}> | undefined) => {
+  if (kql != null) {
+    return toElasticsearchQuery(fromKueryExpression(kql), null);
+  } else if (filter != null) {
+    return filter;
+  } else {
+    // TODO: Re-visit this error (which should never happen) when we do signal errors for the UI
+    throw new TypeError('either kql or filter should be set');
+  }
+};
```

Comment:
I think you added `getFilter` back by mistake during a conflict merge? It is not being referenced or used anymore and the newer one in `get_filter.ts` should be used instead.

Reply:
Yup this was leftover from rebasing with master. Apologies.